src/views/cic/enter-name.html

```diff
@@ -77,17 +79,16 @@ <h1 id="header" class="gov" data-page="{{hmpoPageKey}}" class="govuk-heading-l">
       {% endcall %}
     </div>
 
-    <div>
-      {{ govukDetails({
-        id: "characterDetails",
-        classes: "govuk-!-margin-bottom-7",
-        summaryText: translate("nameEntry.details.summary"),
-        html: detailsBody
-      }) }}
-    </div>
 
     {{ hmpoSubmit(ctx, {classes: "govuk-!-margin-top-0", attributes: {"data-testid": "enter-name-continue-btn"}, text: translate("buttons.next")}) }}
-
+          <div>
+            {{ govukDetails({
+            id: "characterDetails",
+            classes: "govuk-!-margin-bottom-7",
+            summaryText: translate("nameEntry.details.summary"),
+            html: detailsBody
+            }) }}
+          </div>
   {% endcall %}
```

Comment:
We shouldn't be making changes to the default page as part of this ticket - also this change will cause the same incorrect layout mentioned below